### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by abcdefghijohn**
Added stub to admin test

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -26,6 +26,7 @@ group :development, :test do
   gem 'pry'
   gem 'simplecov'
   gem 'shoulda-matchers'
+  gem 'mocha'
```
Comment (jakeheft) [inline Gemfile]: Remove mocha gem

**Review by GregJMitchell — APPROVED** [review]
Coded In tandem with John

File: app/controllers/welcome_controller.rb [hunk 2]
```diff
@@ -2,4 +2,8 @@ class WelcomeController < ApplicationController
   def index
     
   end
+<<<<<<< HEAD
 end
+=======
+end
+>>>>>>> 81e05bbafd607042e66b71a51bb58abff6a6fec2
```
Comment (jakeheft) [inline app/controllers/welcome_controller.rb]: Remove this

File: spec/features/navigation/admin_spec.rb [hunk 3]
```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'mocha'
```
Comment (jakeheft) [inline spec/features/navigation/admin_spec.rb]: Remove mocha

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -27,6 +51,7 @@
          <%= link_to "All Users", "/admin/users" if current_user.role == "admin" %>
        <p><%= "Logged in as #{current_user.name}"  %></p>
      <% end %>
+>>>>>>> 81e05bbafd607042e66b71a51bb58abff6a6fec2
```
Comment (jakeheft) [inline app/views/layouts/application.html.erb]: Remove this

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -10,8 +15,27 @@
...
+          <%= link_to "Cart: #{cart.total_items}", "/cart" if current_user.role != "admin" %>
+          <%= link_to "My Profile", "/profile" if current_user.role == 0 || 1 || 2 %>
+          <%= link_to "Log Out", "/logout", method: :delete if current_user.role == 0 || 1 || 2 %>
+          <%= link_to "Merchant Dashboard", "/merchant" if current_user.role == "merchant" %>
+          <%= link_to "Admin Dashboard", "/admin" if current_user.role == "admin" %>
+          <%= link_to "All Users", "/admin/users" if current_user.role == "admin" %>
+        <p><%= "Logged in as #{current_user.name}"  %></p>
+      <% end %>
+      
+=======
```
Comment (jakeheft) [inline app/views/layouts/application.html.erb]: Remove this

File: app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -10,8 +15,27 @@
 
   <body>
     <nav class = "topnav">
+      <%= link_to "Home", "/"%>
       <%= link_to "All Merchants", "/merchants"%>
       <%= link_to "All Items", "/items"%>
+<<<<<<< HEAD
```
Comment (jakeheft) [inline app/views/layouts/application.html.erb]: Remove this

**Review by jakeheft — APPROVED** [review]
Everything else looks great

**Review by GregJMitchell — APPROVED** [review]

**Review by GarrettCottrell — APPROVED** [review]

**Pull request merged**